README.md

```diff
@@ -1,216 +1,65 @@
+- patches_legacy -> legacy patches directory in which code for the patches where put
+- patches -> used for matrix-js-sdk patches
+- src -> code containing ex matrix-react-sdk lib and element-web code
 
-# config.json
+## Local dev installation
 
-Element supports a variety of settings to configure default servers, behaviour, themes, etc.
-See the [configuration docs](docs/config.md) for more details.
+```
+yarn install
+yarn start
 
-# Labs Features
+```
 
-Some features of Element may be enabled by flags in the `Labs` section of the settings.
-Some of these features are described in [labs.md](https://github.com/element-hq/element-web/blob/develop/docs/labs.md).
+## Dev guidelines
 
-# Caching requirements
+### Making a change
 
-Element requires the following URLs not to be cached, when/if you are serving Element from your own webserver:
+- In element code :
+  You need to add around your code those comments
 
 ```
-/config.*.json
-/i18n
-/home
-/sites
-/index.html
+// :TCHAP: NAME_OF_THE_PATCH
+...code
+// end :TCHAP:
 ```
 
-We also recommend that you force browsers to re-validate any cached copy of Element on page load by configuring your
-webserver to return `Cache-Control: no-cache` for `/`. This ensures the browser will fetch a new version of Element on
-the next page load after it's been deployed. Note that this is already configured for you in the nginx config of our
-Dockerfile.
-
-# Development
-
-Please read through the following:
-
-1. [Developer guide](./developer_guide.md)
-2. [Code style](./code_style.md)
-3. [Contribution guide](./CONTRIBUTING.md)
+Then also update the `tchap-modifications.json` file. We continue to keep track of the changes we make to the sdk. It will also be easier to separate different functionnality that tchap added to the code
 
-# Translations
+### Tests
 
-To add a new translation, head to the [translating doc](docs/translating.md).
-
-For a developer guide, see the [translating dev doc](docs/translating-dev.md).
-
-# Triaging issues
-
-Issues are triaged by community members and the Web App Team, following the [triage process](https://github.com/element-hq/element-meta/wiki/Triage-process).
-
-We use [issue labels](https://github.com/element-hq/element-meta/wiki/Issue-labelling) to sort all incoming issues.
+- Now that `matrix-react-sdk` is merged inside tchap-web, we only target the `test/tchap` folder in order to run our tests on only the files that tchap has modified.
+- For every modification, we need to copy the existing test (if there is one) of the component, move it to tchap folder and modify it accordingly.
 
 ## Copyright & License
 
-Copyright (c) 2014-2017 OpenMarket Ltd
-Copyright (c) 2017 Vector Creations Ltd
-Copyright (c) 2017-2025 New Vector Ltd
+Copyright (c) 2024-2025 Direction Interministérielle du Numérique - Gouvernement Français
 
-This software is multi licensed by New Vector Ltd (Element). It can be used either:
+This software is multi licensed by DINUM. It can be used either:
```

Comment:
DINUM is not "multilicensing": it has picked up the AGPL version and releasing code from DINUM's repository under a single license.

Let's credit New Vector Ltd (Element) as the author of Element (from which we fork) so that people reading this can check Element's repository and use the code under other licenses if they need to.